src/server/shared/SharedDefines.h

```diff
@@ -3146,9 +3148,10 @@ enum WeatherType
 
 #define MAX_WEATHER_TYPE 4
 
-enum ChatMsg
+// EnumUtils: DESCRIBE THIS
+enum ChatMsg : uint8
 {
-    CHAT_MSG_ADDON                  = 0xFFFFFFFF, // -1
+    CHAT_MSG_ADDON                  = 0xFF, // -1
```

Comment:
are you sure this is used as uint8 the change is and a no-op?

Reply:
In the end it is sent as uint8.

https://github.com/TrinityCore/TrinityCore/blob/f3b7f9ed5cb8fdc8142b95bfd16bdfb4a7b10e26/src/server/game/Chat/Chat.cpp#L523